detox/e2e/support/utils/email.ts

```diff
@@ -151,7 +151,7 @@ export const getWelcomeEmailTemplate = (siteUrl, userEmail, siteName, teamName)
         '',
         'Download ( https://mattermost.com/download/#mattermostApps )',
         '',
-        '© 2021 Mattermost, Inc. 530 Lytton Avenue, Second floor, Palo Alto, CA, 94301',
+        '© 2022 Mattermost, Inc. 530 Lytton Avenue, Second floor, Palo Alto, CA, 94301',
```

Comment:
Will this change every year? How is this affecting mobile e2e?

Reply:
Yeah, it's the same in cypress testing

Reply:
won't be best to set that as a variable that matches the current year instead of having to be updating it every year? this seems a bit flaky

Reply:
Updated the file to have dynamic year

Reply:
It's [static on server](https://github.com/mattermost/mattermost-server/blob/master/i18n/en.json#L3268) and very much tied to release version than current year.
Using current year will potentially cause test to fail when running against a server version (eg. ESR) that's using different year value.

Reply:
@saturninoabril I think it's ok for the test to fail, at least for the case that it's a mismatch on the year, which can be used to alert the devs to update the string values if they haven't yet. Also, the tests are most likely run against the latest server version, so that should lessen the impact.

Reply:
I'm thinking not about the non-updated year in current server version but we'll run mobile test in lower server for compatibility reason like latest mobile app against ESR and it will potentially fail there.

Reply:
I think we'll have at most 2 cases: current year and last year. I don't suppose we support anything beyond last year. And I suppose it's only esr that has that scenario. Assuming that's correct, then when i create the email tests, I can add a new parameter for the year, which will default to current year, unless overriden in the test based most likely by url check.

However, I'm thinking esr is not always last year and can be same year. So there's already overhead for maintaining esr.

Another approach might just be an environment variable while running the test which overrides current year if specified

Reply:
Awesome, +1 with env variable via `test_config.ts` in setting up year value. Thanks Joe!